aws_s3_rules/aws_s3_access_error.py

```diff
@@ -10,7 +10,9 @@
 
 def rule(event):
     return (fnmatch(event['operation'], 'REST.*.OBJECT') and
-            event['httpstatus'] in HTTP_STATUS_CODES_TO_MONITOR)
+            event['httpstatus'] in HTTP_STATUS_CODES_TO_MONITOR and
+            event['errorcode'] != 'IncompleteBody'
```

Comment:
Do you anticipate this list to grow? should it be another set?

Reply:
When reviewing the alerts for this rule, this is the only one I noticed regularly being alarmed on. For now I don't anticipate it to grow, but if at some point we decide to add more exceptions we can make it into a set.